sherpa/astro/ui/utils.py

```diff
@@ -26,7 +27,6 @@
 import numpy
 
 import sherpa.ui.utils
-from sherpa.astro.data import DataARF
 from sherpa.astro.instrument import create_arf, create_delta_rmf, create_non_delta_rmf
 from sherpa.ui.utils import _argument_type_error, _check_type, _send_to_pager
```

Comment:
Should we "fix" the access to functions that are implied to not be publicly accessible and rename those functions?

Reply:
Sorry - I dont' understand what you mean. Can you give an example?

Reply:
the module imports `_send_to_pager` but the `_` in front of the name suggests this should not be accessed and used outside of the module where it's defined. Same for other functions. It's a minor thing, but since we are there...

Reply:
> We don't have an articulated strategy for naming routines/ I tend to think of `_xxx` routines as ones that "internal" code may use, from other modules even, but shouldn't really be called by the user, or are part of any API or semantic stability we try to keep for "user" routines. I think we would want to come up with some sort of standard here before making large-scale changes.

If we don't have a standard of our own, the language standards should apply. `_*` names should be considered as "private", i.e. implementation details subject to change and not to be used outside of the scope where they are defined.

They should not be imported in other modules (Python prevents them from being imported with a `*` import). I think the current code is sloppy and the `_` should be removed, as clearly the functions are not meant to be private but used outside of the scope where they are defined. I also don't think it's a large scale change. An IDE worth its salt would change this quite easily, and we are talking about three functions not largely used in the code base. I won't let this stop this PR from being merged.